docs/docs/installation/configuring-superset.mdx

```diff
@@ -169,8 +174,9 @@ RequestHeader set X-Forwarded-Proto "https"
 
 ### Custom OAuth2 Configuration
 
-Beyond FAB supported providers (GitHub, Twitter, LinkedIn, Google, Azure, etc), its easy to connect
-Superset with other OAuth2 Authorization Server implementations that support “code” authorization.
+Superset is built on Flask-AppBuilder (FAB), which supports many providers out of the box
```

Comment:
I mentioned FAB = Flask-AppBuilder here so that the reference makes sense later.

Reply:
thanks